plugins/ocm-backend/package.json

```diff
@@ -42,20 +42,23 @@
     "prepack": "backstage-cli package prepack",
     "start": "backstage-cli package start",
     "test": "backstage-cli package test --passWithNoTests --coverage",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "build:api-docs": "./scripts/openapi-doc.sh",
+    "build:api-server": "yarn backstage-repo-tools package schema openapi generate --server"
   },
   "configSchema": "config.d.ts",
   "dependencies": {
     "@backstage/backend-common": "^0.23.3",
+    "@backstage/backend-dynamic-feature-service": "^0.2.15",
+    "@backstage/backend-openapi-utils": "^0.1.15",
```

Comment:
I added this dependency. Do I need to add that to peerDependencies of dist-dynamic package.json?